src/controllers/cohort.js

```diff
@@ -10,3 +16,23 @@ export const create = async (req, res) => {
     return sendMessageResponse(res, 500, 'Unable to create cohort')
   }
 }
+
+/**
+ * Updates a user by ID
+ * @param {import('express').Request} req
+ * @param {import('express').Response} res
+ */
+export const getById = async (req, res) => {
+  // TODO: not User.findAll()
+  const id = parseInt(req.params.id)
+
+  try {
+    const allUsers = await User.findAll()
```

Comment:
what if we have 1000 users, you'll go to DB, get all 1000 rows and then filter out here, better to directly filter before sending the query using the prisma `where` condition.

You could add a `Cohorts` domain file, that has a function `Cohort.getUsers(cohortId)` -> this then internally calls prisma with the correct where clause